Project-3/Peihong_Xie_Project_3_50375781.md

```diff
@@ -0,0 +1,381 @@
+H(Computer) -->|part_of|F(Network)
+H(Computer) -->|is_carrier_of|A(RDF File Content)
+H(Computer) -->|is_carrier_of|D(Turtle seriaization)
+H(Computer) -->|is_carrier_of|E(XML seriaization)
+J(computer 1) -->|enables|M(Data Transmission)
+M(Data Transmission) -->|ends_with|N(Data Reception)
+K(computer 2) -->|participates_in|N(Data Reception)
+L(computer 3) -->|participates_in|N(Data Reception)
+O(network I) -->|instance_of|F(Network)
+J(computer 1) -->|part_of|O(network I)
+K(computer 2) -->|part_of|O(network I)
+L(computer 3) -->|part_of|O(network I)
+```
+
+This graph represents the following key relations:
+
+– RDF File Content, Turtle and XML are all generically dependent continuants. RDF File Content *has_model* in Turtle and XML. And they are all *carried by* Computer.
+
+– Computer 1 *enables* the process of Data Transmission, which *ends_with* another process, Data Reception, which has computers 2 and 3 as *participants*. So this part is intended to show that there is some data (Turtle and XML) which are sent from Computers 1 and 2 to Computer 3.
+
+– Finally, computers 1, 2, and 3 are all *parts_of* network I, which is *instance_of* Object Aggregate.
+
+**[6]** Using Protege, place these in the BFO hierarchy where you think they fit best:
+```
+  (a) Bach's Well-Tempered Clavier
+  (b) Chair of the UB Philosophy Department
+  (c) SARS-CoV-2
+  (d) Mexico City
+  (e) The trunk of a minivan
+  (f) Occupation
+  (g) Ocean
+  (h) Lake
+```
+
+**Answer:** 
+
+(a) Bach's Well-Tempered Clavier is_a Generically Dependent Continunat.
+
+(b) Chair of the UB Philosophy Department is_a Role.
+
+(c) SARS-CoV-2, as a disease, is_a Disposition.
+
+(d) Mexico City is_a Site.
+
+(e) The trunk of a minivan is_a Fiat Object Part.
+
+(f) Occupation is_a Role.
+
+(g) Ocean is_a Site.
+
+(h) Lake is_a Site.
+
+**[7]** True or False; explain your answers:
+```
+  (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+  (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+  (c) An organization may have another organization as part.
+  (d) An organization may have no members as part. 
+  (e) Any site is partially bounded by some instance of Material Entity.
+  (f) A book placed under the leg of a wobbly table has acquired a new function. 
+  (g) A glass vase cushioned with packing tape for all time, has the disposition to break. 
+  (h) Spacetime is a class in BFO.
+  (i) The continuant fiat boundary class of BFO is closed, meaning, there are no subclasses beyond those identified presently in BFO. 
+```
+
+**Answer:**
+
+(a) True. Boundary or sites are in some cases **continuant parts** of their material hosts (Arp, Smith & Spear 2015: 108). For example, a digestive tube can have its *bona fide* boundary (biological membrane) as part.  
+
+(b) False. According to its definition in BFO, any immaterial entity contains no material entities as parts (Arp, Smith & Spear 2015: 107).
```

Comment:
This is fine as it is. It might be a good idea to use a concrete example and explain why it wouldn't work.